cultivation/scripts/flashcore/cli/_vet_logic.py

```
@@ -0,0 +1,149 @@
+yaml.indent(mapping=2, sequence=4, offset=2)
+yaml.preserve_quotes = True
+
+
+def yaml_to_string(data: Dict[str, Any]) -> str:
+    """Dumps YAML data to a string."""
+    string_stream = StringIO()
+    yaml.dump(data, string_stream)
+    return string_stream.getvalue()
+
+
+def _process_single_file(file_path: Path, check: bool) -> Tuple[bool, bool]:
```

Pylint (3.3.7)

[refactor] Too many local variables (21/15)

(R0914)